Cake.Recipe/Content/parameters.cake

```diff
@@ -459,7 +459,7 @@ public static class BuildParameters
         WyamTheme = wyamTheme ?? "Samson";
         WyamSourceFiles = wyamSourceFiles ?? "../../" + SourceDirectoryPath.FullPath + "/**/{!bin,!obj,!packages,!*.Tests,}/**/*.cs";
         WebHost = webHost ?? string.Format("{0}.github.io", repositoryOwner);
-        WebLinkRoot = webLinkRoot ?? title;
+        WebLinkRoot = webLinkRoot ?? RepositoryName;
```

Comment:
Just a thought, shouldn't we perhaps have the fallback being set to CI provider repository name instead, and if that is null fall all the way back to the `RepositoryName` variable.

For instance:

```suggestion
        WebLinkRoot = webLinkRoot ?? BuildProvider.Repository?.Name ?? RepositoryName;
```

Reply:
Hmm, now that we have this information, that I a good suggestion.  @nils-a what do you think?  If you agree, would you mind putting in a PR to adjust?

Reply:
Sure, but.. I don't think this is "good" :-)

I *think* it should be 

```suggestion
        RepositoryName = repositoryName ?? BuildProvider.Repository?.Name ?? Title;
        WebLinkRoot = webLinkRoot ?? RepositoryName;
```

(this does not work so goog using the *suggestions* - `RespositoryName` should be modified in that case, should it not?)

Reply:
@nils-a I think that this is a sensible suggestion.  @AdmiringWorm thoughts?  In either case, I was thinking about this change, and I think we need to update the docs: https://cake-contrib.github.io/Cake.Recipe/docs/fundamentals/set-parameters#weblinkroot @nils-a would you be in a position to update this?

Reply:
will do.